README.md

```diff
@@ -1,12 +1,17 @@
 # Terraform Instaclustr Provider
 
-A [Terraform](http://terraform.io) provider for managing Instaclustr Platform resources.  
+[![CircleCI](https://img.shields.io/circleci/build/gh/instaclustr/terraform-provider-instaclustr/master?style=for-the-badge)](https://app.circleci.com/pipelines/github/instaclustr/terraform-provider-instaclustr)
```

Comment:
(question) badge is still showing build as failing ?  On release we'd expect a passing badge

Reply:
The badge is showing failing because it doesn't reflect builds of tags. Our releases are based off tags, so nothing runs on master. On merge of this PR however it will run the tests on master branch which will be reflected on the badge